cli/index.ts

```diff
@@ -715,14 +724,28 @@ export function runCli() {
         positionalOptions: [projectDirMustExistOption],
         options: [trackOption],
         processFn: async argv => {
+          let warehouse: string | undefined;
+          try {
+            const dataformJson = fs.readFileSync(path.resolve(argv[projectDirMustExistOption.name], "dataform.json"), 'utf8');
+            const projectConfig = JSON.parse(dataformJson);
+            warehouse = projectConfig.warehouse;
+          } catch {
+            throw new Error('Could not parse dataform.json');
```

Comment:
include the thrown error message?

Reply:
Does this fix look good?
70338e0a06

Reply:
no need to use `ErrorWithCause` unless you really want to; I think the message is enough in this case!

Reply:
I've reverted it & include only the message.
https://github.com/dataform-co/dataform/pull/1490/commits/9a613148c4258fcf6e50041506e41a5f1d53eb9f